contracts/asset/Asset.sol

```diff
@@ -0,0 +1,32 @@
+// SPDX-License-Identifier: AGPLv3
+pragma solidity ^0.8.4;
+
+import "../lists/ScoreList.sol";
+import "../dao/Dao.sol";
+import "./AssetERC20.sol";
+
+contract Asset is AssetERC20, ScoreList, Dao {
+  constructor(
+    address fractionalNFT,
+    uint256 nftID,
+    uint256 shares,
```

Comment:
Assuming shares = the number of tokens to distribute - this would ideally be something that could be set upon "unlocking" via the owner providing their Fractional NFT Key.

Reply:
Easy enough to move over. Will do.

Reply:
Actually, sorry, moving it would break the flow. Locking it in is as simple as sending the NFT, via *any* wallet interface. Moving it requires making a separate function and two transactions to lock it in, instead of just one. I can't recommend this in good faith. Anyone who plans on locking in a NFT should also have plans for shares anyways.